graph.js

```diff
@@ -166,7 +166,7 @@ app
     let logger = new Logger(0, { verbosity: getVerbosity(app) })
 
     if (accessToken === undefined || nodeUrl === undefined || accessToken.length > 200) {
-      console.error('Failed to set access token')
+      console.error('Information missing to set the access token')
```

Comment:
If accessToken is too long this error message may not help quickly identify the issue.

Reply:
Ah, yeah, `missing` is not the best message here. I'll update it.

Reply:
Done